.nais/redis/redisexporter.yaml

```diff
@@ -14,9 +14,6 @@ spec:
     min: 1
     max: 1
   resources:
-    limits:
-      cpu: 100m
-      memory: 100Mi
```

Comment:
Jeg tror vi må beholde `memory`: https://nav-it.slack.com/archives/C5KUST8N6/p1685083616554329?thread_ts=1685083371.794519&cid=C5KUST8N6

Reply:
Ah, akkurat - skal fikse.